be_repo/tests/test_e2e.py

```diff
@@ -0,0 +1,85 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.support import expected_conditions as EC
+import time
+
+FRONTEND_URL = "http://localhost:3001"
+API_URL = "http://127.0.0.1:5001"
+
+options = Options()
+# options.add_argument("--headless")
+driver = webdriver.Chrome(options=options)
+driver.get(FRONTEND_URL)
+wait = WebDriverWait(driver, 10)
+
+try:
+    WebDriverWait(driver, 10).until(
+        EC.presence_of_element_located((By.CLASS_NAME, "nsm7Bb-HzV7m-LgbsSe-MJoBVe"))
+    )
+
+    # Google login
+    google_login_button = driver.find_element(By.CLASS_NAME, "nsm7Bb-HzV7m-LgbsSe")
```

Comment:
This class name is not constant, since the calculating process will be changed if there are changes on frontend DOM Tree. So you'd better to use other selector, like parent and sibling selector, nevigating to elements. @Stanford997 
https://www.w3schools.com/css/css_selectors.asp

Reply:
fixed